packaging/installer/methods/manual.md

```diff
@@ -6,225 +6,120 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/instal
 
 # Install Netdata on Linux from a Git checkout
 
-To install the latest git version of Netdata, please follow these 2 steps:
+These instructions will guide you through building Netdata locally, from a checkout of our GitHub repository. They are primary of
+interest to developers and contributors. Normal users are strongly encouraged to instead use [our official install
+script](./kickstart.md).
 
-1.  [Prepare your system](#prepare-your-system)
+## Prerequisites
 
-    Install the required packages on your system.
+-   git
 
```

Comment:
I would argue that this is redundant given the title of the document. It’s relatively reasonable to assume that anybody who is reading it can figure out that they need `git` if they are installing from a git checkout.

Reply:
I will delete it only for you :D